Sming/SmingCore/Network/WebsocketClient.cpp

```diff
@@ -99,10 +100,11 @@ void WebsocketClient::onError(err_t err)
 
 bool WebsocketClient::_verifyKey(char* buf, int size)
 {
-	String dd = String(buf);
-	uint8_t s = dd.indexOf("Sec-WebSocket-Accept: ");
+	String keyHeader("Sec-WebSocket-Accept: ");
+	String dd(buf);
+	uint8_t s = dd.indexOfIgnoreCase(keyHeader);
```

Comment:
Don't add new method(s) for something that can be achieved with the existing ones. For example you can do something like this:

```
bool WebsocketClient::_verifyKey(char* buf, int size)
{
	String keyHeader("sec-websocket-accept: ");
        String headerValue(buf);
	String headerValueLC = headerValue;
	headerValueLC.toLowerCase();
	int startPos = headerValueLC.indexOf(keyHeader);
	if(startPos == -1) {
		return false;
	}

        // ...
```